src/getting-started/create-bootable-media/bootable-usb-using-macos.md

```diff
@@ -1 +1,99 @@
-Users creating installation media from MacOS can use Etcher. Etcher is an open source startup disk creator capable of writing ISO images to portable storage media.
\ No newline at end of file
+# Create a Bootable Pop!\_OS USB in macOS
+
+This section explains the process of creating a bootable Pop!_OS USB in macOS using balena Etcher.
+
+- [Install ISO flashing software.](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#install-etcher)
+- [Download the Pop!\_OS.iso file.](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#download-the-pop_osiso-file)
+- [Verify the Pop!\_OS.iso Hash Value](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#verify-the-hash-value)
+- [Flash the ISO to your USB storage device.](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#create-the-bootable-usb)
+
+---
+
+## Install Etcher
+
+1. Navigate to Etcher's [download page](https://www.balena.io/etcher/) and click `Download for macOS`.
+
+![Download Etcher](/images/create-bootable-usb-macos/download-etcher.png)
+
+2. Click the download folder from the Dock and select the Etcher .dmg file.
+
+![Run Etcher dmg](/images/create-bootable-usb-macos/run-etcher-dmg.png)
+
+3. Drag the Etcher logo into the applications folder.
+
+![Install Etcher](/images/create-bootable-usb-macos/install-etcher.png)
+
+## Download the Pop!_OS.iso File
+
+![Linux Download ISO](/images/create-bootable-usb-linux/using-linux-download-iso.png)
+
+1. Download the appropriate Pop!_OS.iso file for your target system. See the [preparation guide](/Getting-Started/Create-bootable-media/create-bootable-usb.html#choose-a-pop_os-image) for information about each version.
+
+2. Take note of the hash value for your chosen ISO. Use this value to verify the integrity of the ISO file.
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
+
+3. If prompted, confirm you want to allow downloads from pop.system76.com.
+
+## Verify the Hash Value
+
+1. Click the Launchpad icon, then search for `terminal`. Click the `Terminal` icon to launch it.
+
+![Launch Terminal](/images/create-bootable-usb-macos/launch-terminal.png)
+
+2. Change directory to the folder containing your Pop!\_OS ISO file. In this example the ISO is in the Downloads folder.
+
+```bash
+cd Downloads
+```
+
+3. Use the `shasum` command to verify your ISO file. Make sure the name of the ISO matches exactly what is in your folder. The filename changes depending on the ISO version you chose.
```

Comment:
`changes` is present-tense and `chose` is past-tense; I would go with `chooses` to make them consistent. (Could also reword `changes` to something else instead.)

Reply:
I can change the wording, but I'm not sure I understand the verb-tense suggestion, wouldn't "The filename you use for this step varies depending on the ISO version you chose in the previous step" still be correct?

Reply:
I typo'd my suggestion, I meant to say `choose` and not `chooses`.

You are correct, I don't think there is anything _technically_ wrong with the sentence, or at least it's technically identical to other sentences that sound normal. The way it's currently written just read weirdly to me. "varies" is more passive than "changes"; "changes" almost sounds like it's happening right in front of your eyes, while "varies" just suggests it may be different based on the situation.

Just changing `changes` to `varies` in the sentence as you wrote it would make it sound better in my opinion, if you'd be okay with that change. If you feel strongly that it would negatively impact the tone, you could ignore this suggestion.